include/functions.php

```diff
@@ -201,3 +201,37 @@ function pll_is_plugin_active( string $plugin_name ) {
 
 	return in_array( $plugin_name, $plugins );
 }
+
+/**
+ * Prepares and registers errors
+ *
+ * Wraps `add_settings_error()` to make its use more consistent.
+ *
+ * @since 3.6
+ *
+ * @param WP_Error $error Error object.
+ * @return void
+ */
+function pll_add_settings_error( WP_Error $error ) {
```

Comment:
As we don't use them only for settings, what do you think to rename the function? Maybe `pll_add_error_notice()` ?

Reply:
Renamed to `pll_add_notice()` in bee5b32d5d80cfa3233f596a38193a992be6429c